src/mame/layout/segaai.lay

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="touchpadcursor">
+		<image state="1">
+			<data><![CDATA[
```

Comment:
If the cursor is always supposed to be visible you don’t need the `state="1"` here.  It’s needed for BEENA because there are two areas the pen can target (book page and tablet area).